com.googlecode.cppcheclipse.core/src/com/googlecode/cppcheclipse/core/command/CppcheckCommand.java

```diff
@@ -297,14 +297,14 @@ public static Problem parseResult(String line, IProject project) {
 		try {
 
 			File filename;
-			if (Strings.isNullOrEmpty(lineParts[0])) {
+			if (Strings.isNullOrEmpty(lineParts[0]) || lineParts[0].equals(new String("nofile"))) {
```

Comment:
any reason why "nofile".equals(lineParts[0]) is not good enough (that will never throw an NPE and does not require calling the string constructor)?

Reply:
Not really.. but if it works its also fine by me... i change/test this.

Reply:
Please take another look at it..
Other question.. when is a new version released (including this fix if accepted)